read-fonts/Cargo.toml

```diff
@@ -22,6 +22,7 @@ font-types = { version = "0.5.5", path = "../font-types", features = ["bytemuck"
 serde = { version = "1.0", features = ["derive"], optional = true }
 core_maths = { workspace = true, optional = true }
 bytemuck = { workspace = true }
+int-set = { path = "../int-set" }
```

Comment:
Just a note that we can’t currently take this dependency in read-fonts. This will prevent releasing and break Chrome builds. 

The cmap closure code that makes use of it is relatively simple and only uses public APIs so I’d recommend just moving it to klippa.

Reply:
oh, thanks! I will move it to klippa. I'm taking next Mon and Tue off, if it's not urgent, I will try to do this next Wed.

Reply:
No worries! I don’t think we have any need to release soon but if we do, this is a simple patch so we can take care of it. Enjoy your time off!